src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -75,14 +77,23 @@ public Controls(Subsystems s) {
 		if (DRIVEBASE_ENABLED && LAUNCHER_ENABLED && INTAKE_ENABLED) {
 			// temporary controls, not sure what drive team wants
 			driveController
-					.leftBumper()
+					.leftTrigger()
 					.whileTrue(
 							new FullTargetCommand(
 									s.launcherSubsystem,
 									s.intakeSubsystem,
 									s.drivebaseSubsystem,
 									this,
-									driveController.rightBumper()));
+									driveController.leftTrigger()));
```

Comment:
If we're switching to triggers (which I don't suggest), this needs to be changed to rightTrigger or some other button, otherwise FullTargetCommand will launch as soon as it begins. This parameter describes the button that, when pressed, will launch the note

Reply:
if you want i can switch it to bumpers, or ask drive team for a preference

Reply:
currently waiting on drive team's response

Reply:
Driveteam was fine with it, changed to bumper.